task1/main.c

```diff
@@ -0,0 +1,91 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+#define maxLengthForBinary 256
```

Comment:
```suggestion
#define MAX_LENGTH_FOR_BINARY 256
```
Капсом нужно define константы писать
(-0.1)